src/main/webapp/WEB-INF/pages/main.jsp

```diff
@@ -21,9 +21,14 @@
                     </a>
                     <div class="progress">
                         <div class="progress-bar progress-bar-success progress-bar-striped" role="progressbar"
-                             aria-valuenow="{{project.percendDonate}}" aria-valuemin="0"
+                            aria-valuenow="{{project.percentApprovedDonate}}" aria-valuemin="0"
+                            aria-valuemax="100"
+                            ng-style="{'width':project.percentApprovedDonate +'%'}">
+                        </div>
+                        <div class="progress-bar progress-bar-warning progress-bar-striped" role="progressbar"
+                             aria-valuenow="{{project.donateSubtractApprovePercent}}" aria-valuemin="0"
                              aria-valuemax="100"
-                             ng-style="{'width':project.percendDonate +'%'}">
+                             ng-style="{'width':project.donateSubtractApprovePercent +'%'}">
```

Comment:
create some method to calculate this value and name it something like `pendingDonationPersent`